src/formatDate.js

```diff
@@ -50,7 +50,55 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat[3];
+  const separator = toFormat[3];
+  const arrDate = date.split(oldSeparator);
+  let day;
+  let month;
+  let year;
```

Comment:
it's a good pracitce to initialize new values with falsy value - string: `""`, number: `0`, object: `{}` etc.